ir-gen-cli-bundler/conjureircli/run.go

```diff
@@ -24,11 +24,16 @@ import (
 	"runtime"
 
 	"github.com/mholt/archiver"
-	conjureircli_internal "github.com/palantir/godel-conjure-plugin/v6/ir-gen-cli-bundler/conjureircli/internal"
+	"github.com/palantir/godel-conjure-plugin/v6/ir-gen-cli-bundler/conjureircli/internal"
 	"github.com/palantir/pkg/safejson"
 	"github.com/pkg/errors"
 )
 
+var (
+	//go:embed internal/conjure.tgz
+	conjureCliFS embed.FS
```

Comment:
Is there a reason that `embed.FS` is used here rather than having the variable be a `[]byte` so that the bytes are stored directly? I'm guessing it may have to do with the archiver API, but wanted to verify.

Reply:
Good question; not sure why I did not notice that capability before. Updated!